.gitignore

```diff
@@ -14,5 +14,5 @@ output
 .idea
 .DS_Store
 .tox
-.venv
+venv
```

Comment:
.venv is a commonly used location for virtualenvs. Maybe it's worth ignoring `venv` as well, but not instead of.

Reply:
Must have slipped in.